src/test/java/uk/nickbdyer/tictactoe/GameTest.java

```diff
@@ -5,10 +5,13 @@
 import uk.nickbdyer.tictactoe.players.PlayerFactory;
 import uk.nickbdyer.tictactoe.players.MockUI;
 
+import java.util.Arrays;
+
 import static junit.framework.TestCase.assertTrue;
 import static org.junit.Assert.assertEquals;
 import static uk.nickbdyer.tictactoe.Mark.O;
 import static uk.nickbdyer.tictactoe.Mark.X;
```

Comment:
You could import all marks:
`import static uk.nickbdyer.tictactoe.Mark.*;`
(same was as the `PlayerType`s)